fibbingnode/misc/mininetlib/iprouter.py

```diff
@@ -98,6 +99,29 @@ def __init__(self, router, *args, **kwargs):
         self.ospf.redistribute.static = 1000
         self.ospf.router_id = router.id
 
+        # Parse LSA throttling parameters
+        delay = CFG.get("DEFAULT", 'delay')
+        delay = delay if delay != '' else 0
```

Comment:
Allright :+1: I just wanted to make sure that the default values apply in case nothing is specified in default.cfg